client/rest-high-level/src/main/java/org/opensearch/client/tasks/TaskInfo.java

```diff
@@ -250,6 +262,8 @@ public String toString() {
             + headers
             + ", resource_stats="
```

Comment:
I'm noticing this is not `camelCase` like others. Maybe @sgup432 you can followup on that? It's used in a log so maybe we can just change it?